includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -1043,8 +1040,10 @@ protected function prepare_payment_information( $order ) {
 			$payment_information->must_save_payment_method_to_store();
 		}
 
+		$session_data = WC()->session->get( 'platform-checkout-user-data' );
```

Comment:
```suggestion
		$session_data = WC()->session->get( Platform_Checkout_Extension::PLATMORM_CHECKOUT_SESSION_KEY );
```
I can't get enough of this constant!